rosidl_cmake/cmake/rosidl_generate_interfaces.cmake

```diff
@@ -145,7 +145,10 @@ macro(rosidl_generate_interfaces target)
   set(rosidl_generate_interfaces_IDL_FILES ${_idl_files})
   set(rosidl_generate_interfaces_DEPENDENCY_PACKAGE_NAMES ${_recursive_dependencies})
   set(rosidl_generate_interfaces_SKIP_INSTALL ${_ARG_SKIP_INSTALL})
-  ament_execute_extensions("rosidl_generate_interfaces")
+  if(_ARG_TRACE)
+    set(_trace_args "TRACE")
+  endif()
+  ament_execute_extensions("rosidl_generate_interfaces" ${_trace_args})
```

Comment:
The function `ament_execute_extensions` does not accept any other arguments. So the changes from this file should be removed.